### DISCUSSION TITLE
Close ou Dispose?

### DISCUSSION THREAD

**Discussion by pedro-mileipp** [balta-io/forum#188] [Tira Dúvidas] [answered]
- Boa noite, caríssimo! Poderia explicar melhor a diferença entre  `connection.Close();`  e `connection.Dispose();` ?

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @pedro-mileipp, como vai?
Eu explico esta diferença já no primeiro módulo do curso de Dapper:
https://balta.io/player/assistir/63c6ef55-44ee-445d-9b53-62e06fc969b8
